src/Settings.cpp

```diff
@@ -6,6 +6,9 @@
  */
 
 #include <OsqpEigen/Settings.hpp>
+#include <iostream>
+
+template <typename... Args> inline void unused(Args&&...) {}
```

Comment:
Why this function is required?

Reply:
To avoid the warning about unused parameters when the corresponding setting is not available.

Reply:
Genius!

Reply:
Not really, I have found it somewhere in StackOverflow 😂 The nice thing is that you can pass multiple parameters separated by a comma.